Update how-to-set-clientoptions-in-go.md (#1409)
temporalio/documentation: docs/go/how-to-set-clientoptions-in-go.md

- Type: `TrafficController`

> ### Interceptors

Reviewer: Should this heading also be styled as `inline code`?

> Grpc interceptors which will be applied to every RPC call performed by this connection. By default, an interceptor will be included which automatically retries retryable errors. If you do not wish to perform automatic retries, set this to an empty list (or a list with your own interceptors).

Reviewer: ```suggestion
gRPC interceptors that are applied to every RPC call performed by this connection.
By default, an interceptor is included; it automatically retries retryable errors.
If you do not want to perform automatic retries, set this to an empty list (or a list with your own interceptors).
```
